teflo_webhooks_notification_plugin/webhooks_notification_plugin.py

```diff
@@ -140,6 +149,6 @@ def __init__(self, notification):
 
         super(GchatNotificationPlugin, self).__init__(notification=notification)
 
-        self.url = self.url = self.creds_params.get('gchat_url', None)
```

Comment:
Was it a typo here before?

Reply:
yes this was a typo before, i guess it got overlooked as it did not caused any errors. it looks like a copy paste issue by me :)